src/main/resources/tags/util/logs.tag

```diff
@@ -3,5 +3,7 @@ aliases: log, mclogs
 
 ---
 
-Please share your *full* logs by uploading to https://mclo.gs/ and sharing the link after you have saved it. All IP addresses will be hidden.
-For BungeeCord there is a proxy.log.0 file in the root folder and for everything else it is latest.log in logs folder.
+An entire log file is a very useful way of debugging any issues with plugins. Sometimes a snippet of a log doesn't tell the whole story!
+
+You can upload your log file into this channel, or you can use https://mclo.gs/ to censor all IPs present in your log.
```

Comment:
> You can upload your log file into this channel

The bot doesn't work for paste.gg and paste.gg uploads have been failing a bit too often recently :/